cmd/protoc-gen-goclay/genhandler/template.go

```diff
@@ -203,17 +203,30 @@ func addValueTyped(f *descriptor.Field) string {
 		if valueFormatter != "" {
 			return fmt.Sprintf(valueFormatter, getter)
 		}
+
+		if f.GetProto3Optional() {
```

Comment:
is it possible to have both value formatter and optional for the same field?
I guess not, but could you please check it?

Reply:
We can have both -- currently, the valueformatter is used for encoding bytes. Bytes evaluate to the byte[] scalar type on [go](https://developers.google.com/protocol-buffers/docs/proto#scalar) which is nullable. So the current implementation works.

The question is do we want to create pointers for bytes[] on code generation instead? And if yes, why?

Reply:
Agree. I don't think we need a pointer to a slice